src/tools/perf/lib/uct_tests.cc

```diff
@@ -506,8 +506,7 @@ class uct_perf_test_runner {
                     /* Wait until getting ACK from responder */
                     sn = get_recv_sn(recv_sn, m_perf.uct.recv_mem.mem_type);
                     ucs_assertv(UCS_CIRCULAR_COMPARE8(send_sn - 1, >=, sn),
-                                "recv_sn=%d iters=%" PRIu64, sn,
-                                m_perf.current.iters);
+                                "recv_sn=%d iters=%lu", sn, m_perf.current.iters);
```

Comment:
"iters" is uint64_t, so better use PRIu64
can we include `<inttypes.h>` as a fix?

Reply:
@yosefe Both the customer and I tried this approach before but it didn't help.